CONTRIBUTING.md

```diff
@@ -81,7 +81,8 @@ codebase follows these rules, things are work in progress.
 7) **Always use `iox::log::Logger`**, instead of `printf()`
 8) **Always use `iox::ErrorHandler()`**, instead of the direct STL calls
```

Comment:
What are those direct STL calls and how can the error handler help me there? This is poorly explained and sounds like the error handler would be some substitute which it is not.

We use the error handler once a error that cannot or shall not be propagated via expected has occurred, as was (still is?) described in error-handling.md. STL uses a totally different strategy, namely exceptions. Using error the handler is not related to STL but to any calls where errors can occur (POSIX calls with return codes etc.).